### ISSUE TITLE
When I set 2 data-targets to an toggle-button containing data-toggle="collapse", class collapsed cannot be added properly after the button is actived. 

### ISSUE THREAD

**Issue by zhou9197** [twbs/bootstrap#21482]
As title, 
1. if I set one object (#sidebar) to data-target, class collapsed will be added after I click the button.
<a class="navbar-brand **_collapsed_** " data-toggle="collapse" data-target='#sidebar'>
   <span class="glyphicon glyphicon-indent-right"></span>
</a>
2. if I set two objects (#sidebar, #content) to data-target, class collapsed will **_NOT_** be added after I click the button.
<a class="navbar-brand **_ NO collapsed_** " data-toggle="collapse" data-target='#sidebar, #content'>
   <span class="glyphicon glyphicon-indent-right"></span>
</a>
3. why I set 2 data-targets. 
class 'in' will be toggled to #sidebar, #content, with which I toggle the sidebar and reset the margin-left of #content using css .XX.in{display:xx; margin-left:xx}
4. why I need class 'collapse' for toggle button
 I'd like to use css .collaspse.glyphicon-indent-right{content: '\XXX'} to toggle glyphicon-right or left
5. Could you help to provide any method that meets my need without any additional javascript?

**Comment by mdo** [comment]
Duplicate report of #17643 I think, at least for v4.

**Comment by zhou9197** [comment]
hi bro
I am using bootstrap v3 not v4. And both targets can be toggled. That's not the question I ask about.
I mean `collapse in ` can be added and removed to the toggleE but `collapseD` can  not be added to toggleR if I set 2 data-targets like `#navbar #content`. However the toggleR can be toggled with class `collapseD` if I set only one data-target. It is not the same issue with #17643. Did you misunderstand me?
